plugin/trino-redis/src/test/java/io/trino/plugin/redis/TestRedisConnectorTest.java

```diff
@@ -41,50 +42,20 @@ public void destroy()
     }
 
     @Override
-    protected boolean supportsCreateSchema()
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     {
-        return false;
-    }
-
-    @Override
-    protected boolean supportsCreateTable()
-    {
-        return false;
-    }
-
-    @Override
-    protected boolean supportsInsert()
-    {
-        return false;
-    }
+        switch (connectorBehavior) {
+            case SUPPORTS_CREATE_SCHEMA:
```

Comment:
Is it ok to group all of these here, but not in `TestCassandraLatestConnectorSmokeTest.java` and others?

Reply:
i was lazy

Reply:
fixed